### PR TITLE
rev to M17; update the changelog

### PR DESCRIPTION
**Pull request by devoncarew**
- rev to M17 (https://github.com/flutter/flutter-intellij/issues/1248)
- update the changelog

@stevemessick @mit-mit @pq

### CODE REVIEW

**Comment by devoncarew** [comment]
(TBR)

File: resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -22,6 +22,19 @@
   <change-notes>
     <![CDATA[
 
+17.0:
+<ul>
+  <li>improved new project wizard</li>
+  <li>improvements to the reload-on-save behavior</li>
+  <li>improved and reorganized the Flutter view's toolbar</li>
+  <li>fixes to the flutter icon decorations in the editor ruler</li>
```
Comment (mit-mit) [inline resources/META-INF/plugin.xml]: Should capitalise `flutter` here

File: resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -22,6 +22,19 @@
...
+  <li>fixes to group handling for widget tests</li>
+  <li>display a ballon toast if there are analysis issues wen running apps</li>
+  <li>added a toggle inspect mode toolbar button</li>
+  <li>added an experimental preference to show the (WIP) memory view</li>
```
Comment (mit-mit) [inline resources/META-INF/plugin.xml]: I think we should leave this out until we are ready to have people actually use it.

File: resources/META-INF/plugin.xml [hunk 3]
```diff
@@ -22,6 +22,19 @@
...
     <![CDATA[
 
+17.0:
+<ul>
+  <li>improved new project wizard</li>
+  <li>improvements to the reload-on-save behavior</li>
+  <li>improved and reorganized the Flutter view's toolbar</li>
+  <li>fixes to the flutter icon decorations in the editor ruler</li>
+  <li>fixes to group handling for widget tests</li>
+  <li>display a ballon toast if there are analysis issues wen running apps</li>
```
Comment (mit-mit) [inline resources/META-INF/plugin.xml]: s/wen/when/

**Pull request merged**